nautobot/extras/management/__init__.py

```diff
@@ -389,3 +395,82 @@ def _clear_custom_role_or_status_instances(
 
     if verbosity >= 2:
         print(f"    Deleted {deleted_total}, unlinked {unlinked_total} {metadatamodel} records")
+
+
+def validate_job_and_job_data(command, user, job_class_path, data=None):
+    job_data = {}
+    try:
+        if data:
+            job_data = json.loads(data)
```

Comment:
Do we need to use `NautobotKombuJSONEncoder` here by any chance? Thinking about things like datetime and UUID handling.

Reply:
We did not have NautobotKombuJSONEncoder originally in the `runjob` code. But I can definitely add it.

Reply:
Should be fine as-is then. Thanks.